src/main/java/luke/Storage.java

```diff
@@ -0,0 +1,49 @@
+package luke;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class Storage implements Serializable {
```

Comment:
Considering future iterations of Luke, it may be beneficial to create a separate class, such as TxtStorage, that inherits from this file. This approach allows for potential expansion beyond storing data solely in text files in future versions of the application. Just a suggestion.